python/kfserving/requirements.txt

```diff
@@ -12,3 +12,4 @@ adal >= 1.2.2
 table_logger >= 0.3.5
 numpy
 azure
+azure-storage-blob == 1.3.0
```

Comment:
@rakelkar do we need the entire `azure` or just `azure-storage-blob` ?

Reply:
@rakelkar @ellis-bigelow can we pin a lot of these versions? its a bit dangerous to just leave these without versions and have them self-resolve to the most recent. It causes errors like #513 to come up

Reply:
Does this mean azure is making backwards incompat changes? =/

Reply:
Yes they did indeed make a breaking change :( 
I pinned like you did here and removed the dependency on azure as well here: https://github.com/kubeflow/kfserving/pull/518